pkgs/development/libraries/ogdf/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, doxygen }:
+
+stdenv.mkDerivation rec {
+  pname = "ogdf";
+  version = "2020.02";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "catalpa-202002";
+    sha256 = "0drrs8zh1097i5c60z9g658vs9k1iinkav8crlwk722ihfm1vxqd";
+  };
+
+  hardeningDisable = [ "all" ];
```

Comment:
Can you provide a more specific list of disabled hardening options? If you really sure that it must be "all", then can you, perhaps, add some comment explaining why you think this is the most appropriate?

Reply:
Yes, I weakened it from `all` to `format`, and added a comment on where this became relevant. 
Thanks for checking this.